ActiveLabeler-main/ActiveLabelerModels/Linear_models.py

```diff
@@ -0,0 +1,85 @@
+from torch import nn
+
+
+class SSLEvaluator(nn.Module):
+    """The classification model which is used with the SSL encoder"""
+
+    def __init__(self, n_input, n_classes, n_hidden=512, p=0.1, c_type="Sigmoid"):
+        super().__init__()
+        """
+        Keyword arguments
+        n_input -- Size of the input embeddings
```

Comment:
Hungarian notation is typically discouraged for variable naming. Consider changing these to `input_size`, `num_classes` or `class_count`, etc.